Thing.h

```diff
@@ -696,9 +692,11 @@ class ThingDevice {
   }
 
   void serialize(JsonObject descr, String ip, uint16_t port) {
-    descr["id"] = this->id;
+    descr["id"] = "uri:" + this->id;
```

Comment:
So that it actually is a valid uri. The spec requires it to be a valid URI and not some random ID.
https://w3c.github.io/wot-thing-description/#thing